Lib/fontTools/misc/py23.py

```diff
@@ -1,525 +1,148 @@
-"""Python 2/3 compat layer."""
+"""Python 2/3 compat layer leftovers."""
 
-from __future__ import print_function, division, absolute_import
-import sys
+import decimal as _decimal
+import math as _math
+import warnings
+from contextlib import redirect_stderr, redirect_stdout
+from io import BytesIO
+from io import StringIO as UnicodeIO
+from types import SimpleNamespace
+
+warnings.warn(
+    "The py23 module has been deprecated and will be removed in the next release. "
+    "Please update your code.",
+    DeprecationWarning,
+)
+
+__all__ = [
+    "basestring",
+    "bytechr",
+    "byteord",
+    "BytesIO",
+    "bytesjoin",
+    "open",
+    "Py23Error",
+    "range",
+    "RecursionError",
+    "round",
+    "SimpleNamespace",
+    "StringIO",
+    "strjoin",
+    "Tag",
+    "tobytes",
+    "tostr",
+    "tounicode",
+    "unichr",
+    "unicode",
+    "UnicodeIO",
+    "xrange",
+    "zip",
+]
 
 
-__all__ = ['basestring', 'unicode', 'unichr', 'byteord', 'bytechr', 'BytesIO',
-		'StringIO', 'UnicodeIO', 'strjoin', 'bytesjoin', 'tobytes', 'tostr',
-		'tounicode', 'Tag', 'open', 'range', 'xrange', 'round', 'Py23Error',
-		'SimpleNamespace', 'zip', 'RecursionError']
+class Py23Error(NotImplementedError):
+    pass
 
 
-class Py23Error(NotImplementedError):
-	pass
-
-
-PY3 = sys.version_info[0] == 3
-PY2 = sys.version_info[0] == 2
```

Comment:
you may as well keep these PY3 and PY2 constants, there may be code around still using them (can't remember)

Reply:
actually scrap that, it's ok to remove.